user/customizing-the-build.md

```diff
@@ -523,12 +524,27 @@ jobs:
 ```
 {: data-file=".travis.yml"}
 
+#### Conditionally Allowing Jobs to Fail
+
+Allowed failures can include a [condition](/user/conditional-builds-stages-jobs#conditionally-allowing-jobs-to-fail) using the key `if`.
+
+For example, the following would allow the job using `rvm: 1.9.3` to fail
+only on the `master` branch:
+
+```yaml
+jobs:
+  allow_failures:
+  - rvm: 1.9.3
+    if: branch = master
+```
+{: data-file=".travis.yml"}
```

Comment:
Since this is already documented elsewhere, is it a good idea to duplicate the information here? (You know, stuff getting desynced with time, users getting confused, everyone's time and nerves getting wasted.)

I'd go with only a reference: "A job [can be allowed to fail conditionally](https://docs.travis-ci.com/user/conditional-builds-stages-jobs#conditionally-allowing-jobs-to-fail)"